src/01-Learning-JS/07-dates.md

```diff
@@ -371,11 +381,21 @@ Here are some tips to consider as you complete this exercise.
 <p class="tip"><strong>Isolate interested parts of data</strong>: If dates are important to the inquiry, remember that the dates are stored as Strings in the following format: <code>"10/24/2024"</code>, i.e.,<code>"mm/dd/YYYY"</code>.</p>
 
 <!-- E1 -->
-```javascript
+```js
 // Convert and code here
+const slashDateFormat = utcFormat("%m/%d/%Y")
```

Comment:
Ah, a little issue of using the wrong method. You are parsing, rather than formatting.

```suggestion
const slashDateParser = utcParse("%m/%d/%Y")
```